comps/text2cypher/src/integrations/cypher_utils.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import re
+from collections import namedtuple
+from string import Template
+from typing import Any, Dict, List, Optional, Union
+
+from langchain_community.chains.graph_qa.cypher_utils import CypherQueryCorrector, Schema
+
+from comps import CustomLogger
+
+logger = CustomLogger("opea_text2cypher_cypher_utils")
+Relationship = namedtuple("Relationship", ["entity1", "relationship", "entity2"])
+
+cypher_cleanup = """
+ MATCH (n) DETACH DELETE n
+"""
+
+cypher_insert = """
+ LOAD CSV WITH HEADERS FROM 'https://docs.google.com/spreadsheets/d/e/2PACX-1vQCEUxVlMZwwI2sn2T1aulBrRzJYVpsM9no8AEsYOOklCDTljoUIBHItGnqmAez62wwLpbvKMr7YoHI/pub?gid=0&single=true&output=csv' AS rows
```

Comment:
Just summarize some key points from our discussion offline:

1) offer customizable cypher_insert statements (e.g. in the Input class in comps/text2cypher/src/integrations/native.py), passed by users.
2) provide the capability to cleanup previous database when users input new cypher_insert input (may be by setting a flag in the Input class).
3) add README about above changes.